cg/meta/workflow/raredisease.py

```diff
@@ -22,6 +29,125 @@ def __init__(
         pipeline: Workflow = Workflow.RAREDISEASE,
     ):
         super().__init__(config=config, pipeline=pipeline)
+        self.root_dir: str = config.raredisease.root
+        self.nfcore_pipeline_path: str = config.raredisease.pipeline_path
+        self.references: str = config.raredisease.references
+        self.profile: str = config.raredisease.profile
+        self.conda_env: str = config.raredisease.conda_env
+        self.conda_binary: str = config.raredisease.conda_binary
+        self.config_platform: str = config.raredisease.config_platform
+        self.config_params: str = config.raredisease.config_params
+        self.config_resources: str = config.raredisease.config_resources
+        self.tower_binary_path: str = config.tower_binary_path
+        self.tower_pipeline: str = config.raredisease.tower_pipeline
+        self.account: str = config.raredisease.slurm.account
+        self.compute_env: str = config.raredisease.compute_env
+        self.revision: str = config.raredisease.revision
+
+    def config_case(
+        self,
+        case_id: str,
+        dry_run: bool,
+    ) -> None:
+        """Create config files (parameters and sample sheet) for Raredisease analysis."""
+        self.create_case_directory(case_id=case_id, dry_run=dry_run)
+        sample_sheet_content: list[list[Any]] = self.get_sample_sheet_content(case_id=case_id)
+        pipeline_parameters: PipelineParameters = self.get_pipeline_parameters(case_id=case_id)
+        if dry_run:
+            LOG.info("Dry run: Config files will not be written")
+            return
+        self.write_sample_sheet(
+            content=sample_sheet_content,
+            file_path=self.get_sample_sheet_path(case_id=case_id),
+            header=RarediseaseSampleSheetEntry.headers(),
+        )
+        self.write_params_file(case_id=case_id, pipeline_parameters=pipeline_parameters.dict())
+
+    def get_sample_sheet_content_per_sample(
```

Comment:
A unit test for this and the following function would be nice. They are the core of the proposed change

Reply:
Tests added